README.md

```diff
@@ -1,14 +1,166 @@
+
+- ### Function calls
+
+    Basic functions of the low-cost temperature/humidity sensors have been implemented in this library. There's no need to re-implement these functions from scratch. The user simply has to import the library in the project and can use any of its functions by just calling it.
+
+- ### Give back
+
+    The library is free, you don’t have to pay for anything. However, if you want to support the development, or just thank the author of the library by purchasing products from Adafruit!
+
+    Not only you’ll encourage the development of the library, but you’ll also learn how to best use the library and probably some C++ too
+
+- ### MIT License
+
+    DHT sensor library is open-source and uses one of the most permissive licenses so you can use it on any project.
+
+  - Commercial use
+  - Modification
+  - Distribution
+  - Private use
+
+## Functions
+
+- begin()
+- readTemperature()
+- convertCtoF()
+- convertFtoC()
+- readHumidity()
+- computeHeatIndex()
+- read()
+- expectPulse()
+
+## Example
+
+Examples include both a "standalone" DHT example and one that works along with the Adafruit Unified Sensor Library. A Unified sensor library is required even if using the standalone version. You can find other examples from [Github-DHT-sensor-library](https://github.com/adafruit/DHT-sensor-library/tree/master/examples).
+
+```Cpp
+#include "DHT.h"
+
+#define DHTPIN 2
+#define DHTTYPE DHT22
+DHT dht(DHTPIN, DHTTYPE);
+
+void setup() {
+  Serial.begin(9600);
+  Serial.println(F("DHTxx test!"));
+
+  dht.begin();
+}
+
+void loop() {
+  delay(2000);
+
+  float h = dht.readHumidity();
+  float t = dht.readTemperature();
+  float f = dht.readTemperature(true);
+
+  if (isnan(h) || isnan(t) || isnan(f)) {
+    Serial.println(F("Failed to read from DHT sensor!"));
+    return;
+  }
+
+  float hif = dht.computeHeatIndex(f, h);
+  float hic = dht.computeHeatIndex(t, h, false);
+
+  Serial.print(F("Humidity: "));
+  Serial.print(h);
+  Serial.print(F("%  Temperature: "));
+  Serial.print(t);
+  Serial.print(F("°C "));
+  Serial.print(f);
+  Serial.print(F("°F  Heat index: "));
+  Serial.print(hic);
+  Serial.print(F("°C "));
+  Serial.print(hif);
+  Serial.println(F("°F"));
+}
+```
+
+## Contributing
+
+If you want to contribute to this project:
+
+- Report bugs and errors
+- Ask for enhancements
+- Create issues and pull requests
+- Tell others about this library
+- Contribute new protocols
+
+Please read [CONTRIBUTING.md](https://github.com/adafruit/DHT-sensor-library/blob/master/CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests to us.
+
+## Credits
+
+The author and maintainer of this library is Adafruit <info@adafruit.com>
+
+Based on previous work by:
+
+- T. DiCola
+- P. Y. Dragon
+- L. L. Fried
```

Comment:
No middle name, just L. Fried

Reply:
Thanks, @ladyada  for reviewing. I have just fixed this one :)